nnpdf_data/nnpdf_data/commondata/ATLAS_WJ_8TEV/metadata.yaml

```diff
@@ -1,121 +1,116 @@
 setname: ATLAS_WJ_8TEV
-version: 1
-version_comment: Port of old commondata
+
 nnpdf_metadata:
   nnpdf31_process: DY CC
   experiment: ATLAS
+
 arXiv:
   url: https://arxiv.org/abs/1711.03296
-  journal: JHEP 05 (2018) 077
+  journal: JHEP 05 (2018) 07, JHEP 10 (2020) 048 (Erratum)
 iNSPIRE:
-  url: ''
+  url: https://inspirehep.net/literature/1635273
 hepdata:
-  url: ''
-  version: -1
+  url: https://www.hepdata.net/record/ins1635273
+  version: 2
+
+version: 2
+version_comment: Implementation in the new format
+
 implemented_observables:
 - observable_name: WM-PT
   observable:
     description: Jet Transverse Momentum Distribution
     label: ATLAS $W^-$+jet 8 TeV
-    units: ''
-  process_type: EWJ_PT
-  tables: []
-  npoints: []
+    units: '[fb]'
+  process_type: DY_CC_PT
+  tables: [13, 14]
   ndata: 16
   plotting:
-    kinematics_override: ewj_pt_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS $W^-$+jet 8 TeV
     x_label: $p_T^{W^-}$ (GeV)
     y_label: $d\sigma/dp_T^{W^-}$ (fb/GeV)
     y_scale: log
-    plot_x: k1
-  kinematic_coverage:
-  - k1
-  - k2
-  - k3
+    plot_x: pT
+  kinematic_coverage: [pT, m_W2]
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
+      pT:
+        description: Transverse momentum of the W boson
+        label: $p_T^{W^{-}}$
+        units: GeV
+      m_W2:
+        description: W boson mass squared
+        label: $M_W^2$
+        units: GeV$^2$
     file: kinematics_WM-PT.yaml
-  theory:
-    conversion_factor: 1.0
-    operation: 'null'
-    FK_tables:
-    - - ATLAS_WM_JET_8TEV_PT-atlas-atlas-wjets-arxiv-1711.03296-xsec003
-  data_uncertainties: []
+  data_central: data_WM-PT.yaml
+  data_uncertainties: [uncertainties_WM-PT.yaml]
   variants:
+    CMS_prescription:
+      data_uncertainties:
+      - uncertainties_WM-PT_cms.yaml
+      data_central: data_WM-PT_cms.yaml
     legacy:
       data_uncertainties:
       - uncertainties_legacy_WM-PT.yaml
     legacy_NP:
       data_uncertainties:
-      - uncertainties_WM-PT_sys_NP.yaml
+      - uncertainties_legacy_WM-PT_sys_NP.yaml
```

Comment:
No, central data has not changed. Note that now the default variant has symmetrised uncertainties, meaning that central data is shifted by the symmetrisation prescription. On the other hand, the `CMS_prescription` variant does not require any shift to the central data, and for this variant central data matches with the legacy implementation. But you're right - I need to specify in the metadata file that these legacy variants should not use the shifted data version.

In that respect, I have a technical question. As I said, the symmetrisation prescription introduces a shift in the data. Now, this prescription must be applied to any source of uncertainty, so that for the same point I end up with as many shifts as the number of uncertainty sources. I took for granted that all the shifts are simply added to the central data, namely

$$
d_i^{shifted} = d_i + \sum_{k=0}^{sources} s^{(k)}_i
$$

Am I right? @scarlehoff  @enocera